test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/040_ATF_P_TC_default_app_PTU_retry_sequence_finish_ForceProtectedService_OFF_RPC_encrypted_true.lua

```diff
@@ -11,11 +11,10 @@
 -- SDL must respond StartService (ACK, encrypted=false) to this mobile app
 --
 -- 1. Used preconditions:
--- RPC SetAudioStreamingIndicator is allowed by policy
 -- ForceProtectedService is set to OFF in .ini file
 -- Default app exists in LP, no certificate in module_config
 -- Register and activate application.
--- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- Send StartService(serviceType = 7 (RPC))
```

Comment:
@istoimenova, Please remove "SDL should not process request to HMI" statement in line 20